README.md

```diff
@@ -19,3 +19,34 @@ This repository contains machine-readable specifications for the auxillilary ser
 - [ ] Exhaustive RPC testing for Forest docker image.
 - [ ] Sync testing for PRs.
 - [ ] Export testing for PRs.
+
+## Running Forest In DigitalOcean
+
+## Forest IAC Architectue
+
+![Forest Cloud Infrastructure ](https://user-images.githubusercontent.com/47984109/216006502-eca661d3-2ef8-4c75-aa7a-1740c25abb44.png)
+
+The flow goes from Terraform for provisioning of the servers and Anisble to run all neccessary installations including forest.
+
+## Requirements 
+- RAM: 32GB
+- VCPU: 8
+- Startup Disk Size: 200 GB
+- Expected Total Disk Size: > 500 GB
+
+N/B: It's worth to note that most of the naming conventions can be changed to suit your deployment needs especially the names for the resource blocks. In this case; `forest-volume`, `new-key-name`,`spaces-name`,`forest-volume`, `forest`, `forest-firewalls-test` and `hosts`.
+
+To test out the implementation, just access the server with appropiate `ssh` details in this manner `ssh -i ~/.ssh/id_rsa chainsafe@ip_address`. However, if you intend to use another `user` other than `chainsafe`, this can be switched in the Ansible main playbook, just avoid running forest as root.
+
+In order ro implement the infrastructure, run the following:
+- Set Up s3cmd 2.x with DigitalOcean Spaces; you can check [here](https://docs.digitalocean.com/products/spaces/reference/s3cmd/) on more details. This will require `ACCESS_TOKEN` and `SECRET_KEY` but it can be automatically generated from the DigitalOcean console through the Applications & API section.  
```

Comment:
```suggestion
- Set-up `s3cmd` 2.x with DigitalOcean Spaces; you can check [here](https://docs.digitalocean.com/products/spaces/reference/s3cmd/) on more details. This will require `ACCESS_TOKEN` and `SECRET_KEY`; it can be automatically generated from the DigitalOcean console through the Applications & API section.  
```